src/index.css

```diff
@@ -112,3 +112,8 @@ table > :last-child > :last-child > :last-child {
     @apply bg-background text-foreground;
   }
 }
+
+.mdxeditor {
+  border-radius: 5px;
+  background-color: white;
+}
```

Comment:
we could just shift these styling rules to be passed as props since `MarkdownEditor` takes in an optional `className` prop

this makes it easier for theme switching integration moving forward